supervisor/store/__init__.py

```diff
@@ -83,6 +83,12 @@ async def _add_repository(url: str, step: int):
                 await repository.load()
             except StoreGitError:
                 _LOGGER.error("Can't load data from repository %s", url)
+                self.sys_resolution.create_issue(
+                    IssueType.CORRUPT_REPOSITORY,
+                    ContextType.STORE,
+                    reference=repository.slug,
+                    suggestions=[SuggestionType.EXECUTE_REMOVE],
+                )
```

Comment:
Create on git pull error this SuggestionType.EXECUTE_REMOVE instead of the current one. We don't need create an issue on that place

Reply:
https://github.com/home-assistant/supervisor/blob/main/supervisor/store/git.py#L114-L118